go/engine/paperprovision.go

```diff
@@ -167,6 +162,11 @@ func (e *PaperProvisionEngine) paper(ctx *Context, kp *keypair) error {
 		if err := e.makeDeviceKeysWithSigner(ctx, kp.sigKey); err != nil {
 			return err
 		}
+		if err := e.G().LoginState().Account(func(a *libkb.Account) {
+			a.SetUnlockedPaperKey(kp.sigKey, kp.encKey)
+		}, "UnlockedPaperKey"); err != nil {
+			return err
+		}
```

Comment:
Hmm.  I was hoping this was going to be one line:

    lctx.SetUnlockedPaperKey(....)

But that method isn't currently in LoginContext.  The above LoginState() block isn't going to work as this code is already in a serialized LoginState process.

Options:

1. Add `SetUnlockedPaperKey` to LoginContext interface and use the line above.
2. Put this block back where you had it.

I'd prefer option 1.